lib/kcerts/ssh.go

```diff
@@ -219,6 +294,19 @@ func FindSSHAgentFromEnv() *SSHAgent {
 // CreateNewSSHAgent creates a new ssh agent. Its env variables have not been added to the shell. It does not maintain
 // its own connection.
 func CreateNewSSHAgent() (*SSHAgent, error) {
+	a := &SSHAgent{}
+
+	socket, err := a.GetStandardSocketPath()
+	if err != nil {
+		return nil, err
+	}
+
+	os.Remove(socket) // ignore errors
```

Comment:
this can also go? now that's a symlink? eg, ssh-agent will use a random path anyway, so no reason to check if the symlink/old path is in place here?